### ISSUE TITLE
Разработка на хосте

### ISSUE THREAD

**Issue by sircha** [GyverLibs/GyverPortal#95]
Не совсем понимаю как запустить скомпилированную прошивку локально, не заливая ее в микроконтроллер. Можно более подробную инструкцию?

**Comment by GyverLibs** [comment]
вообще без понятия)

**Comment by sircha** [comment]
я про этот пункт "Примеры можно скомпилировать и выполнить на хосте, запустив ../../onhost [-h] из любого каталога примеров."

**Comment by GyverLibs** [comment]
да. Но человек, который предложил данную возможность - слишком умный, и не дал больше никаких инструкций. Сказал что и так всем всё понятно

**Comment by aromprg** [comment]
Инструкцию смотрите тут https://github.com/GyverLibs/GyverPortal/pull/62#issuecomment-1528813919
@GyverLibs можно добавить в вики

**Comment by GyverLibs** [comment]
@aromprg  ага спасибо добавил
